From 6266f3c37ac428259289e85cff88d168aac9f139 Mon Sep 17 00:00:00 2001
From: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
Date: Fri, 22 Nov 2019 09:39:32 +0700
Subject: [PATCH 171/351] arm64: defconfig: Enable kernel module for USB Gadget
 Function

This patch add default build kernel module for USB Gadget Function.

Signed-off-by: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
---
 arch/arm64/configs/defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index d2484fa..9bc5c52 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -368,3 +368,6 @@ CONFIG_DEVFREQ_GOV_USERSPACE=y
 CONFIG_DEVFREQ_GOV_PASSIVE=y
 CONFIG_E1000E=y
 CONFIG_RENESAS_MFIS_ECC=y
+CONFIG_USB_MASS_STORAGE=m
+CONFIG_USB_G_SERIAL=m
+CONFIG_USB_ETH=m
-- 
2.7.4

